backend/README.md

```diff
@@ -31,3 +31,46 @@ Hello world!
 ```
 
 Done!
+
+# To get Docker running 🐳
```

Comment:
Thank you for this! Ran through it and it seems like the connection is ok 🐳 🚀 🕺 !